### ISSUE TITLE
Typo in entry for "Anti-Regeneration Ray" (id 44040)

### ISSUE THREAD

**Issue by onyxavenger** [zzorba/marvelsdb#247]
Card "Anti-Regeneration Ray" has a typo in its textbox. This is card id 44040. https://marvelcdb.com/card/44040

The Forced Interrupt text reads "[...] _threat_ the attacked character's text box [...]" (emphasis added). That word should be "treat" based on the card text from the image. I noticed this on mouseover a link for the card, and confirmed it on the page linked above. I assume the mouseover text pulls from this entry, but if not it may need to be corrected in both locations.

**Comment by mmougy** [comment]
https://github.com/zzorba/marvelsdb-json-data/pull/530

**Comment by onyxavenger** [comment]
Reviewed the change in code; this should resolve the issue once implemented. It does not currently seem to be complete on marvelcdb.com, so I assume that this code will be added in the future. I will close issue once I can confirm it's been done, unless there is some other process I should be doing.

**Comment by mmougy** [comment]
> Reviewed the change in code; this should resolve the issue once implemented. It does not currently seem to be complete on marvelcdb.com, so I assume that this code will be added in the future. I will close issue once I can confirm it's been done, unless there is some other process I should be doing.

Indeed, this code will be reviewed and maybe merged when administrators could
